app/frontend/tests/unit/store/modules/form.actions.spec.js

```diff
@@ -9,12 +9,14 @@ import {
 } from '~/services';
 import { useFormStore } from '~/store/form';
 import { useNotificationStore } from '~/store/notification';
+import { useAppStore } from '../../../../src/store/app';
```

Comment:
Could this use the alias:
`import { useAppStore } from '~/store/app';`

Reply:
ha, yeah. that will be better and not look so gnarly.